numba/tests/test_extending.py

```diff
@@ -239,6 +241,28 @@ def clip_usecase(x, lo, hi):
     return x.clip(lo, hi)
 
 
+# -----------------------------------------------------------------------
+
+def return_non_boxable():
+    return np
+
+
+@overload(return_non_boxable, jit_options={'no_cpython_wrapper': True})
```

Comment:
Well... `@overload` compiled functions are not callable from pure Python, so `no_cpython_wrapper` could be the default, no?